interactive_engine/src/executor/store/src/v2/errors.rs

```diff
@@ -16,3 +16,13 @@
 pub enum Error {
     Internal(String),
 }
+
+impl Error {
```

Comment:
better impl Display for Error

Reply:
Add only for print temporarily， also can be removed or refactored later.